extra-dev/packages/coq-hierarchy-builder/coq-hierarchy-builder.dev/opam

```diff
@@ -6,13 +6,20 @@ homepage: "https://github.com/math-comp/hierarchy-builder"
 bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
 dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
 
-build: [ make "build"]
-install: [ make "install"]
-depends: [ "coq-elpi" {= "dev"} ]
-synopsis: "Hierarchy Builder"
+build: [ [ make "build"]
+         [ make "test-suite" ] {with-test}
+       ]
+install: [ make "install" ]
+depends: [ "coq-elpi" { (>= "1.11.0" & < "1.13~") | = "dev" } ]
```

Comment:
```suggestion
depends: [ "coq-elpi" {>= "1.11.0"} ]
```
As usual for `dev` packages, lower bounds are preferred, so we don't have to update so much in the future.